requirements.txt

```diff
@@ -4,4 +4,5 @@ gql[requests]==3.4.1
 pycoingecko==3.1.0
 pandas==2.0.3
 simplejson==3.19.2
-git+https://github.com/BalancerMaxis/bal_addresses@0.9.1
+git+https://github.com/BalancerMaxis/bal_addresses@0.9.4
+git+https://github.com/BalancerMaxis/bal_tools@v0.0.1
```

Comment:
bal tools is at v0.0.3 already.

bump it, or maybe just unpin it?

Reply:
Because we are not using pypi bal_tools either has to not be provided, or the @ string has to match the current version in that version of bal_addresses. bal_addresses@0.9.1 still uses ball tools 0.0.1

Reply:
but you are bumping bal addresses to 0.9.3 here, which uses bal_tools@v0.0.3

Reply:
ah no wait that is not true; main branch is using v0.0.3 but v0.9.4 is still using v0.0.1